app/controllers/users_controller.rb

```diff
@@ -14,7 +14,7 @@ def show
 
   # POST /user
   def create
-    @user = User.new params.require(:user).permit!
```

Comment:
This could be potentially dangerous. `.permit!` essentially whitelists everything. Refactored this to use strong params properly, only whitelisting attributes in the form